src/Pim/Bundle/EnrichBundle/Normalizer/ProductViolationNormalizer.php

```diff
@@ -44,12 +56,19 @@ public function normalize($violation, $format = null, array $context = [])
                 'attribute' => $productValue->getAttribute()->getCode(),
                 'locale'    => $productValue->getLocale(),
                 'scope'     => $productValue->getScope(),
-                'message'   => $violation->getMessage()
+                'message'   => $violation->getMessage(),
+            ];
+        } elseif ('identifier' === $path) {
+            $normalizedViolation = [
+                'attribute' => $this->attributeRepository->getIdentifierCode(),
```

Comment:
I thought the identifier wasn't related to an attribute anymore... That's confusing.